modules/common/src/utils/utils.ts

```diff
@@ -172,6 +173,23 @@ export function requiredEnvVariable (key: keyof typeof process.env): string {
     throw new Error(`Missing environment variable "${key}"!`);
 }
 
+export function requiredEnvProdVariable (key: keyof typeof process.env, envType: EnvironmentType): string | null {
+    if (envType !== EnvironmentType.PROD) {
+        return null;
```

Comment:
should we return it also here if its there? We just require it on prod? so just return the <string> process.env[key];?